### PR TITLE
mopidy-gmusic: init at 1.0.0

### PR DESCRIPTION
**Pull request by jgillich**
This should work in theory, but it seems like mopidy extensions in general aren't working (tested on unstable):

```
mopidy[2776]: INFO     Disabled extension gmusic: Dependency The 'gmusicapi>=6.0' distribution was not found and is required by the application not found
mopidy[2776]: INFO     Disabled extension spotify: Dependency The 'pyspotify<1.999,>=1.9' distribution was not found and is required by the application not found
```

Can anyone look into this? I don't really know where to start.

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -4427,11 +4427,11 @@ in modules // {
...
 
   gmusicapi = with pkgs; buildPythonPackage rec {
-    name = "gmusicapi-4.0.0";
+    name = "gmusicapi-7.0.0";
 
     src = pkgs.fetchurl {
-      url = "https://pypi.python.org/packages/source/g/gmusicapi/gmusicapi-4.0.0.tar.gz";
-      md5 = "12ba66607531978b349c7035c9bab311";
+      url = "https://pypi.python.org/packages/source/g/gmusicapi/gmusicapi-7.0.0.tar.gz";
+      md5 = "bba75437f7f5bf06847c26d154f4324f";
```
Comment (domenkozar) [inline pkgs/top-level/python-packages.nix]: please use `sha256` when bumping packages

**Comment by jgillich** [comment]
@domenkozar Fixed, also updated mopidy, but I'm still getting the above error.

**Comment by jgillich** [comment]
Rebased on master, which fixed the travis build. I think this can be merged. Mopidy extensions with additional dependencies don't work, but that's a separate issue.

**Comment by jgillich** [comment]
Opened #11706

**Comment by spwhitt** [comment]
LGTM!

I've recently made additional mopidy related improvements, which I'll create a PR for soon. This may solve some of the remaining issues.

**Pull request merged**